types.js

```diff
@@ -60,11 +60,8 @@ type InAppBrowserAndroidOptions = {
   hasBackButton?: boolean,
   browserPackage?: string,
   showInRecents?: boolean,
-};
+|};
 
-export type InAppBrowserOptions = {
-  ...InAppBrowserAndroidOptions,
-  ...InAppBrowseriOSOptions
-};
+export type InAppBrowserOptions = InAppBrowserAndroidOptions | InAppBrowseriOSOptions;
```

Comment:
Maybe this line can be reverted 
https://github.com/proyecto26/react-native-inappbrowser/pull/198#discussion_r505698609

Reply:
Hello mate, we need the options from both types, can you fix that?